manifests/4.6/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -35,29 +42,32 @@ spec:
             description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
             properties:
               inputs:
-                description: Inputs are named inputs of log messages
+                description: "Inputs are named filters for log messages to be forwarded.
+                  \n There are three built-in inputs named `application`, `infrastructure`
+                  and `audit`. You don't need to define inputs here if those are sufficient
+                  for your needs. See `inputRefs` for more."
                 items:
                   description: InputSpec defines a selector of log messages.
                   properties:
                     application:
-                      description: 'Enable `application` logs. Use `application: {}`
-                        to enable with no filter.'
+                      description: 'Application enables `application` logs. `application:
+                        {}` enables all application logs.'
                       properties:
                         namespaces:
-                          description: Only collect logs from applications in these
-                            namespaces. If empty, all application container logs will
-                            be collected.
+                          description: Namespaces selects logs only from pods in the
```

Comment:
maybe something like "List of specific namespaces to collect pod logs from. If the list is empty..."

Reply:
+1